db/sql/3_msar_custom_aggregates.sql

```diff
@@ -0,0 +1,127 @@
+/*
+This script defines all the necessary functions to be used for custom aggregates.
+
+Currently, we have the following custom aggregate(s):
+  - msar.peak_time(time): Calculate the 'average time' (interpreted as peak time) for a column.
+
+Refer to the official documentation of PostgreSQL custom aggregates to learn more.
+link: https://www.postgresql.org/docs/current/xaggr.html
+
+We'll use snake_case for legibility and to avoid collisions with internal PostgreSQL naming
+conventions.
+*/
+
+
+CREATE SCHEMA IF NOT EXISTS msar;
+
+CREATE OR REPLACE FUNCTION 
+msar.time_to_degrees(time_ TIME) RETURNS DOUBLE PRECISION AS $$/*
+Convert the given time to degrees (on a 24 hour clock, indexed from midnight).
+
+Examples:
+  00:00:00 =>   0
+  06:00:00 =>  90
+  12:00:00 => 180
+  18:00:00 => 270
+*/
+SELECT EXTRACT(EPOCH FROM time_) * 360 / 86400;
```

Comment:
```suggestion
SELECT EXTRACT(EPOCH FROM time_) / 240;
```
Preload the division for performance here, please; explain it in a code comment or the docstring.